.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "Gitea DevContainer",
+  "image": "mcr.microsoft.com/devcontainers/go:1.20",
+  // installs nodejs into container
+  "features": {
+    "ghcr.io/devcontainers/features/node:1": {
+      "version": "20"
+    }
+  },
+  "customizations": {
+		"vscode": {
+			"settings": {},
+      // same extensions as GitPod
+			"extensions": [
+				"editorconfig.editorconfig",
+        "dbaeumer.vscode-eslint",
```

Comment:
Tab/Space are mixed in this file. Should it be unified?

Reply:
You could add the file to editorconfig lint to prevent such issues in the future.